apps/rolodex/views/partials/directory/search_bar.ejs

```diff
@@ -1,7 +1,12 @@
+<style>
```

Comment:
Recommended to put tailwind class in the input tag, instead of putting custom style like this
https://v2.tailwindcss.com/docs/placeholder-color